crates/host/src/wasmbus/mod.rs

```diff
@@ -1317,7 +1317,7 @@ impl ComponentSpecification {
 
 #[allow(clippy::large_enum_variant)] // Without this clippy complains actor is at least 0 bytes while provider is at least 280 bytes. That doesn't make sense
 enum Claims {
-    Actor(jwt::Claims<jwt::Actor>),
+    Actor(jwt::Claims<jwt::Component>),
```

Comment:
Should we rename this variant to Component while we're here?

Reply:
Ah yes, will rename!

- [ ] rename `Claims::Actor` variant